supporting_scripts/analysis-of-endpoint-connections/src/main/java/de/tum/cit/endpointanalysis/EndpointInformation.java

```diff
@@ -0,0 +1,118 @@
+package de.tum.cit.endpointanalysis;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public class EndpointInformation {
+
+    private String requestMapping;
+
+    private String endpoint;
+
+    private String httpMethodAnnotation;
+
+    private String URI;
+
+    private String className;
+
+    private int line;
+
+    private List<String> otherAnnotations;
+
+    public EndpointInformation() {
+
+    }
```

Comment:
Why do we need an empty constructor?

Reply:
The constructor is needed so that ObjectMapper can deserialize the EndpointInformation from the .json file.